### ISSUE TITLE
tweetlist looses css styling when compiling

### ISSUE THREAD

**Issue by pkellner** [SenchaArchitect/TouchSimpleExamples#3]
Using the 2.1 designer build 678, I did the following:
1)   extracted current version from this project
2)   opened project with designer tweetlist
3)   went through simple package steps and put in my 2.1.0 ST library
4)   added requires for plugins to work (see previous issue)
5)   senchacmd:   sencha app build testing {production does same issue}
6)   run project.  styling from resource tweetlist.css seems to be missing

This one is important to me because I've used the twitter page in a app I'm trying to push to the blackberry store and it closes very soon.

**Comment by philstrong** [comment]
I running through this now but it's very likely it's because it's using the generic index.html instead of one that resembles the app.html

**Comment by pkellner** [comment]
I made the css more specific in my real app and that solved it.  The other issue about adding the requires to the twitter plugins is probably more important.

**Comment by philstrong** [comment]
boy oh boy it must have take a while to figure out those requires you needed to add.

**Comment by philstrong** [comment]
Yeah I'm seeing the css issue.  I spoke with Don today and they're totally reworking the css/sass inclusion system so perhaps these sorts of issues won't occur.  Confused as to why it worked before and not now.

**Comment by philstrong** [comment]
The goal will be to have Architect apply these requires for you without out
you even being aware it's happening.

On Mon, Jan 21, 2013 at 11:33 PM, Peter Kellner notifications@github.comwrote:

> I made the css more specific in my real app and that solved it. The other
> issue about adding the requires to the twitter plugins is probably more
> important.
> 
> From: Phil Strong [mailto:notifications@github.com]
> Sent: Monday, January 21, 2013 8:28 PM
> To: SenchaArchitect/TouchSimpleExamples
> Cc: Peter Kellner
> Subject: Re: [TouchSimpleExamples] tweetlist looses css styling when
> compiling (#3)
> 
> I running through this now but it's very likely it's because it's using
> the generic index.html instead of one that resembles the app.html
> 
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/SenchaArchitect/TouchSimpleExamples/issues/3#issuecomment-12530226>.
> 
>  —
> Reply to this email directly or view it on GitHubhttps://github.com/SenchaArchitect/TouchSimpleExamples/issues/3#issuecomment-12530317.

## 

Phil Strong, Software Architect
_Sencha, Inc._
